### PR TITLE
style(formatting): remove platform from user guide

### PR DESCRIPTION
**Pull request by christinaausley**
Relates to https://github.com/camunda/developer-experience/issues/117.

### CODE REVIEW

File: content/user-guide/camunda-bpm-run.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 
-title: 'Camunda Platform Run'
+title: 'Camunda Run'
```
Comment (akeller) [inline content/user-guide/camunda-bpm-run.md:3]: This should be Camunda 7 Run so we leave room for a C8-flavored Run.

Comment (akeller) [reply content/user-guide/camunda-bpm-run.md:3]: I don't think we need to carry this through the entire doc, but definitely the title.

File: content/user-guide/spring-framework-integration/configuration.md [hunk 2]
```diff
@@ -51,19 +51,19 @@ If you want the process engine to be registered with the Camunda Platform Proces
...
               type="org.camunda.bpm.ProcessEngineService"
-              description="Camunda Platform Process Engine Service"
+              description="Camunda 7 Process Engine Service"
               factory="org.camunda.bpm.container.impl.jndi.ProcessEngineServiceObjectFactory" />
 
     <Resource name="java:global/camunda-bpm-platform/process-engine/ProcessApplicationService!org.camunda.bpm.ProcessApplicationService"
               auth="Container"
               type="org.camunda.bpm.ProcessApplicationService"
-              description="Camunda Platform Process Application Service"
+              description="Camunda 7 Process Application Service"
```
Comment (tasso94) [inline content/user-guide/spring-framework-integration/configuration.md:60]:
```suggestion
              description="Camunda Platform Process Application Service"
```

**Review by tasso94 — CHANGES_REQUESTED** [review]
Two hints. Looks good otherwise 🙂 👍

**Pull request merged**